godot-bindings/src/lib.rs

```diff
@@ -130,7 +130,13 @@ pub use prebuilt::*;
 // ----------------------------------------------------------------------------------------------------------------------------------------------
 // Common
 
-const NEXT_MINOR_VERSION: u8 = 3;
+// List of minor versions with the highest known patch number for each.
+//
+// We could have this just be a list of patch numbers, letting the index be the minor version. However it's more readable to include the
+// minor versions as well.
+//
+// Note that when the patch version `0` then the patch number is not included in Godot versioning, i.e `4.1.0` is displayed as `4.1`.
+const KNOWN_API_VERSIONS: &[(u8, u8)] = &[(0, 4), (1, 4), (2, 2), (3, 0)];
```

Comment:
```suggestion
// Note that when the patch is version `0`, then the patch number is not included in Godot versioning, i.e. `4.1.0` is displayed as `4.1`.
const KNOWN_API_VERSIONS: &[(u8, u8)] = &[(0, 4), (1, 4), (2, 2), (3, 0)];
```
<br>

Furthermore, should this now maybe be called `HIGHEST_PATCH_VERSIONS` or so?